design-library/src/components/BccPagination/BccPagination.vue

```diff
@@ -0,0 +1,103 @@
+<script setup lang="ts">
+import {
+  PaginationEllipsis,
+  PaginationList,
+  PaginationListItem,
+  PaginationNext,
+  PaginationPrev,
+  PaginationRoot,
+} from "radix-vue";
+import { ArrowBackIosNewIcon } from "@bcc-code/icons-vue";
+import { ArrowForwardIosIcon } from "@bcc-code/icons-vue";
+import { toRefs, ref, computed, watchEffect } from "vue";
+import BccSelect from "../BccSelect/BccSelect.vue";
+
+type Props = {
+  items: number;
```

Comment:
I would rename this to something like `totalItems` or just `total` to mirror the Radix names (could do that as well for other props if the Radix name is clear enough). But this is nicer than adding all the items.